enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/OtpService.java

```diff
@@ -91,14 +91,10 @@ public String createOtpCode(OnboardingProcessEntity process, OtpType otpType) th
     public String createOtpCodeForResend(OnboardingProcessEntity process, OtpType otpType) throws OnboardingOtpDeliveryException, OnboardingProcessException {
         final String processId = process.getId();
         // Do not allow spamming by OTP codes
-        Date lastDate = onboardingOtpRepository.getNewestOtpCreatedTimestamp(processId, otpType);
-        int resendPeriod = onboardingConfig.getOtpResendPeriod();
-        Calendar c = GregorianCalendar.getInstance();
-        c.setTime(lastDate);
-        c.add(Calendar.SECOND, resendPeriod);
-        Date allowedDate = c.getTime();
-        if (allowedDate.after(new Date())) {
-            logger.warn("Resend OTP functionality is not available yet for process ID: {}", processId);
+        Date otpLastCreatedDate = onboardingOtpRepository.getNewestOtpCreatedTimestamp(processId, otpType);
```

Comment:
Consider adding `final` (also to the variables below).

Reply:
Fixed